### DISCUSSION TITLE
Just wanted to say thanks for making this Plugin

### DISCUSSION THREAD

**Discussion by gobijan** [yaroslavyaroslav/OpenAI-sublime-text#35] [General]
Hey, I just read the 1 year Anniversary message via Package Control.
I wanted to say thank you for creating this plugin and I'm looking forward to the new additions you'll come up with :)
So again: Thank you for your time. Thank you for your effort. Thank you for the work you put in this.

**Comment by yaroslavyaroslav** [comment]
Thank you for the kind words, this matters a lot!

The day will come and we'll overthrown jetbrains IDE domination 🫡

**Comment by jalsco** [comment]
I totally agree ! Your whole project is awesome and your responsiveness is next level. Thank you so much 🙏🏻
I only wish that I was in a position to be able to have paid more, or to be able to pay monthly. I am sorry that I was only able to give so little at this time. You deserve far more than I contributed. Your work is excellence ! all of it.

**Reply by yaroslavyaroslav** [reply]
Thank you for these words and for the donation! Really appreciate that! Sooner rather than later I'll finish that god damn refactoring that is happening now and plugin will become even more exciting

**Comment by jalsco** [comment]
IS the new refactored project (I know it has been MAJOR MAJOR WORK , ty ty ty 🙏🏻❤)
released yet ??   In github still showing as the last release as October 2024 ?

**Reply by yaroslavyaroslav** [reply]
Not yet, but it's around the corner. Current develop branch now leverages this rust library https://github.com/yaroslavyaroslav/ai_helper. 

It's (develop version) not fully functional yet, but I think I'll made it in par within a week or two. And the rest will be on the ST community to include that as dependency to make it distributable through package control.

**Comment by jalsco** [comment]
That's Great news !
